### DISCUSSION TITLE
Enlarge columns

### DISCUSSION THREAD

**Discussion by yoobi** [patrykandpatrick/vico#654] [Questions]
Hello, it's me again :D (I'm using 2.0.0-alpha.14)

I have an issue of columns size
```kotlin
val data = mutableMapOf(
    "My long title" to 3,
    "Another long title" to 5,
    "This one is short" to 2,
    "Shorty" to 7,
    "title" to 12,
    "long title again" to 10,
    "again long title" to 5,
    "again long title" to 9,
    "looooooooooooonger" to 6,
    "AZERTYHGFDSQXCVB" to 6,
    "loremipsm" to 6,
)
val modelProducer = remember { CartesianChartModelProducer.build() }
val xKey = remember { ExtraStore.Key<Set<String>>() }
LaunchedEffect(true) {
    modelProducer.tryRunTransaction {
        columnSeries { series(data.values) }
        updateExtras { it[xKey] = data.keys }
    }
}

CartesianChartHost(
	scrollState = rememberVicoScrollState(),
	chart = rememberCartesianChart(
	    rememberColumnCartesianLayer(),
	    startAxis = rememberStartAxis(),
	    bottomAxis = rememberBottomAxis(
	        valueFormatter = remember {
	            { index, chartValues, _ -> chartValues.model.extraStore[xKey].elementAt(index.toInt()) }
	        },
	    )
	),
	modelProducer = modelProducer,
	modifier = modifier
)
```
<img width="338" alt="Screenshot 2024-04-09 at 12 51 28" src="https://github.com/patrykandpatrick/vico/assets/17434735/dd8bbaa4-9ade-4fef-af33-35de660f1021">

Is the only solution to have wider columns to override the default ?
```kt 
rememberColumnCartesianLayer(
	columnProvider = ColumnCartesianLayer.ColumnProvider.series(
	    vicoTheme.cartesianLayerColors.map { color ->
	        rememberLineComponent(
	            color,
                    Defaults.COLUMN_WIDTH.dp,
	            Shapes.roundedCornerShape(Defaults.COLUMN_ROUNDNESS_PERCENT),
	        )
	    },
	)
)
```

**Comment by Gowsky** [comment]
Hello, @yoobi. Yes, you have to override the column width. Technically, the zoom also affects the column width, but it wouldn't make sense to override the initial zoom level here. Would you expect a feature that adjusts the column width depending on the label length? There was a similar request, but it concerned the automatic adjustment of the column spacing, and not the column width.

**Reply by yoobi** [reply]
Adjusting the column widths depending on the label might do more harm than good and would still need a max width limit for super long string.
My question might be just an edge case and being able to adjust width as I see fit with the method above should be enough
